cmd/xl-v1-multipart.go

```diff
@@ -1204,5 +792,67 @@ func (xl xlObjects) AbortMultipartUpload(bucket, object, uploadID string) error
 	if !xl.isUploadIDExists(bucket, object, uploadID) {
 		return errors.Trace(InvalidUploadID{UploadID: uploadID})
 	}
-	return xl.abortMultipartUpload(bucket, object, uploadID)
+
+	// Read metadata associated with the object from all disks.
+	partsMetadata, errs := readAllXLMetadata(xl.storageDisks, minioMetaMultipartBucket, uploadIDPath)
```

Comment:
this should be `xl.getDisks()` - NOTE `xl.storageDisks` is only used for testing purposes and will be removed in future.

Reply:
oops, fixed